presto-benchto-benchmarks/src/main/resources/benchmarks/presto/tpch.yaml

```diff
@@ -0,0 +1,37 @@
+datasource: presto
+query-names: presto/tpch/${query}.sql
+runs: 6
```

Comment:
Support for overrides (with a command line switch) would be a good addition to Benchto.